azure/template.json

```diff
@@ -442,14 +453,42 @@
                 }
             }
         },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "[concat('StorageContainer', parameters('containerArray')[copyIndex()].name)]",
+            "type": "Microsoft.Resources/deployments",
+            "condition": "[greater(length(parameters('containerArray')), 0)]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'), 'storage-container.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
```

Comment:
Maybe good to hardcode the publicAccess parameter as None so it is clear

Reply:
Added new commit hard coding it to none.